recipes/deployment.rb

```diff
@@ -11,13 +11,13 @@
   if prefer :database, 'sqlite'
     gsub_file 'Gemfile', /.*gem 'sqlite3'\n/, ''
     add_gem 'sqlite3', group: [:development, :test]
-    add_gem 'pg', group: :production
+    add_gem 'pg', '>= 0.18', group: :production
   end
 end
 
 if prefer :deployment, 'capistrano3'
   say_wizard "installing gems for Capistrano"
-  add_gem 'capistrano', '~> 3.0.1', group: :development
+  add_gem 'capistrano', '>= 3.6.0', group: :development
```

Comment:
@AleksSenkou i now how work `~>` And i think 3.6 and next 3.7 will work correctly with backward compatibility. This is not major version number. For version >= 4.x.x i agree with you but not for 3.x.x